middleware/tracing.go

```diff
@@ -6,8 +6,8 @@ import (
 )
 
 type TracingConfig struct {
-	TracerName  string
-	IDGenerator func() string
+	TracerName  string        //TracerName is the name of the response header where the request id will be.
```

Comment:
comments go above the attribute

also struct needs docstring, entire file does really.

Reply:
not sure i remember touching tracing.go, but alright